runtime/oti/VMHelpers.hpp

```diff
@@ -2099,7 +2102,8 @@ class VM_VMHelpers
 	static VMINLINE bool
 	isContinuationMountedOrConcurrentlyScanned(J9VMContinuation *continuation, bool isGlobalGC)
 	{
-		return isContinuationMounted(continuation) || isConcurrentlyScannedFromContinuationState(continuation->state, isGlobalGC);
+		uintptr_t state = continuation->state;
+		return isContinuationMountedFromContinuationState(state) || isConcurrentlyScannedFromContinuationState(state, isGlobalGC);
```

Comment:
create isContinuationMountedOrConcurrentlyScanned(uintptr_t state, bool isGlobalGC)
and use it here and later in tryWinningConcurrentGCScan